--- nsl2vl.org  2022-08-17 10:08:22.776787819 +0000
+++ nsl2vl      2022-08-17 09:59:54.522910168 +0000
@@ -1,6 +1,7 @@
 #!/bin/bash
 NSLCORE=nslc.exe
 NSLPP=nslpp.exe
+CHIBICC=chibicc
 sname=`basename $0` 
 if [ $# -lt 1 ]; then
 ${NSLCORE} -v
@@ -90,9 +91,9 @@
 esac
 
 if [ $supout -eq 0 ]; then
-echo "${NSLPP} $nestcomment $pparg $name | ${NSLCORE} $exearg -o $outname"
-${NSLPP} $nestcomment $pparg $name | ${NSLCORE} $exearg -o $outname
+echo "${CHIBICC} $nestcomment $pparg $name -E | ${NSLPP} | sed '/^#  1  \s*/d' | ${NSLCORE} $exearg -o $outname"
+${CHIBICC} $nestcomment $pparg $name -E | ${NSLPP} | sed '/^#   1  \s*/d' | ${NSLCORE} $exearg -o $outname
 else
-${NSLPP} $nestcomment $pparg $name | ${NSLCORE} $exearg
+${CHIBICC} $nestcomment $pparg $name -E | ${NSLPP} | sed '/^#   1  \s*/d' | ${NSLCORE} $exearg
 fi
 exit $?
