--- /home/koyaman/Downloads/linux/x86_64/nsl2vl	2021-03-01 16:54:35.000000000 +0900
+++ nsl2vl	2021-11-21 17:11:17.875645145 +0900
@@ -1,6 +1,7 @@
 #!/bin/bash
 NSLCORE=nslc.exe
 NSLPP=nslpp.exe
+NSLPP=cpp
 sname=`basename $0`
 if [ $# -lt 1 ]; then
 ${NSLCORE} -v
@@ -91,8 +92,8 @@
 
 if [ $supout -eq 0 ]; then
 echo "${NSLPP} $nestcomment $pparg $name | ${NSLCORE} $exearg -o $outname"
-${NSLPP} $nestcomment $pparg $name | ${NSLCORE} $exearg -o $outname
+${NSLPP} $nestcomment $pparg $name | sed '/^\s*$/d' | ${NSLCORE} $exearg -o $outname
 else
-${NSLPP} $nestcomment $pparg $name | ${NSLCORE} $exearg
+${NSLPP} $nestcomment $pparg $name | sed '/^\s*$/d' | ${NSLCORE} $exearg
 fi
 exit $?
